src/components/sidebar/useCurrentActivePage.ts

```diff
@@ -0,0 +1,132 @@
+import { useState, useEffect, useCallback, useRef } from 'react';
+import { type FormPage } from '../../types';
+import { scrollIntoView } from '../../utils/form-helper';
+
+interface UseCurrentActivePageProps {
+  pages: FormPage[];
+  defaultPage: string;
+  activePages: string[];
+  evaluatedPagesVisibility: boolean;
+}
+
+interface UseCurrentActivePageResult {
```

Comment:
Adding `Use***` for interface naming makes the naming convention confusing. I think we should just change these 2 to `CurrentActivePageProps` and `CurrentActivePageResult` respectively especially since there not being exported anyways.